openshift/templates/listener/el-github-listener-deployment.yaml

```diff
@@ -0,0 +1,327 @@
+kind: Deployment
```

Comment:
Just curious why we need a Deployment for the eventlistener. Can you walk me through what each file is doing?